google/cloud/bigtable/admin_v2/overlay/services/bigtable_table_admin/client.py

```diff
@@ -301,120 +291,74 @@ def sample_wait_for_consistency():
                 be of type `bytes`.
 
         Returns:
-            google.cloud.bigtable.admin_v2.overlay.types.CheckConsistencyPollingFuture:
-                An object representing a polling future.
-
-                The result type for the operation will be `bool`, and will return True when the
-                consistency check involving the given consistency token returns True.
+            bool:
```

Comment:
This only describes the standard_read_remote_mode. Can you also add the description of data_boost_read_local_writes here? Reference: https://source.corp.google.com/piper///depot/google3/google/bigtable/admin/v2/bigtable_table_admin.proto;l=1205?q=GEnerateConsistencyToken&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental